api-client/src/main/java/com/xing/api/internal/json/ContactPathJsonAdapter.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (С) 2016 XING AG (http://xing.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.xing.api.internal.json;
+
+import android.support.annotation.Nullable;
+
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.JsonReader;
+import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.Moshi;
+import com.squareup.moshi.Types;
+import com.xing.api.data.profile.XingUser;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author serj.lotutovici
+ */
+public final class ContactPathJsonAdapter extends JsonAdapter<List<List<XingUser>>> {
```

Comment:
No tests for this, since it's a temp solution, that will be replaced with a more permanent one.